index.html

```diff
@@ -5,12 +5,67 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
+    <title>Domenico's Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="./style.css">
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <header>
+        <section class="head-bar">
+            <img class="img-logo" src="./images/logo.jpeg" alt="logo-cake"/>
+            <img class="hamburger" src="./images/menu-hamburger.svg" alt="hamburgher-menu"/>
+        
+        <nav>
+            <ul>
+                <li><a href="" Home></a>
+                </li>
+                <li>Our cakes</li>
+                <li>Create your own</li>
+                <li>About Us</li>
+                <li>Contact Us</li>
+            </ul>
+        </nav>
+        </section>
+        <h3>The best cake in town</h3>
+    </header>
+
+    <main>
+        <section>
+            <div class="welcomeHero"></div>
+                <h1>Welcome to our creamy cakes paradise</h1>
+                <p>
+You will be able to indulge yourself into the pleasure that you never experienced before. Be careful, our cakes are addictive!                </p>
+            </div>
+            <div>
+                <img class="imgChocoBig" src="./images/choko_big.jpeg" alt="main cake image"/>
+            </div>
+            <section class="cakeDisp">
```

Comment:
Can you use flexbox here to align the 4 images? 
Also, for class names keep the naming lower case e.g. `cake-disp` (by the way, what is `disp`? Maybe use the full word?)